soccer_pycontrol/src/soccer_pycontrol/soccerbot.py

```diff
@@ -79,13 +80,14 @@ def __init__(self, pose, useFixedBase=False, useCalibration=False):
 
         home = expanduser("~")
         self.body = pb.loadURDF(
-            home + "/catkin_ws/src/soccerbot/bez1_description/urdf/soccerbot.urdf",
+            home
+            + f"/catkin_ws/src/soccerbot/{rospy.get_param('~robot_model', 'bez1')}_description/urdf/{rospy.get_param('~robot_model', 'bez1')}.urdf",
             useFixedBase=useFixedBase,
-            flags=pb.URDF_USE_INERTIA_FROM_FILE,
-            basePosition=[pose.get_position()[0], pose.get_position()[1], Soccerbot.standing_hip_height],
+            flags=pb.URDF_USE_INERTIA_FROM_FILE | (pb.URDF_MERGE_FIXED_LINKS if rospy.get_param("~_merge_fixed_links", False) else 0),
+            basePosition=[pose.get_position()[0], pose.get_position()[1], Soccerbot.torso_height],
             baseOrientation=pose.get_orientation(),
         )
-
+        self.pybullet_offset = pb.getBasePositionAndOrientation(self.body)[0][:2] + (0,)  # pb.getLinkState(self.body, Links.TORSO)[4:6]
```

Comment:
what is (0,)?

Reply:
Single element tuple containing 0. (0,)+(1,) = (0,1)